cwf/gateway/tools/uesim_cli/main.go

```diff
@@ -296,12 +302,37 @@ func createUeConfig(imsi string, auth_key []byte, seq_num uint64) (*protos.UECon
 	if err != nil {
 		return nil, fmt.Errorf("could not generate OPc for subscriber: %s: %s", imsi, err)
 	}
+	ues, err := uecfg.GetMap("subscribers")
```

Comment:
line 243 already grabs something called `subscribers`. Not sure if this can be used again

Reply:
We could refactor trafficgen and add ue commands to use single functions to read uedata from yml file.

Reply:
I will do this refactoring and post a new diff for this.